js/helpers.js

```diff
@@ -371,14 +371,6 @@ Helpers.UpdateHtmlFontAttributes = function ($epubHtml, fontSize, fontObj, callb
                     "rel" : "stylesheet",
                     "type" : "text/css"
                 });
-                if (!('onload' in link[0])) {
-//alert("LINK ONLOAD !?");
-                    var imgTag = $epubHtml[0].ownerDocument.createElement("img");
-                    imgTag.onerror = fontLoadCallback_;
-                    imgTag.src = fontObj.url;
-                } else {
-                    link[0].onload = fontLoadCallback_;
-                }
```

Comment:
PS: with the Resize Sensor approach, of course the second font-face "pass" should be captured (and a pagination refresh should occur), but the end-user will be able to observe the change of font face, which is not desired.